src/main/java/com/openlattice/chronicle/controllers/v2/ChronicleControllerV2.java

```diff
@@ -180,6 +183,22 @@ public void sendMessages(
         messageService.sendMessages( organizationId, messageDetails);
     }
 
+    @RequestMapping(
+            path = { ORGANIZATION_ID_PATH + MESSAGE_PATH + STATUS_PATH },
+            method = RequestMethod.POST )
+    @ResponseStatus( HttpStatus.OK )
+    public void updateMessageStatus(
+            @PathVariable ( ORGANIZATION_ID ) UUID organizationId,
+            HttpServletRequest request
+    ) throws ExecutionException {
```

Comment:
Following up on the feedback on https://github.com/openlattice/chronicle-api/pull/60

```suggestion
    public void updateMessageStatus(
            @PathVariable( ORGANIZATION_ID ) UUID organizationId,
            @RequestParam( MESSAGE_SID ) String messageSid,
            @RequestParam( MESSAGE_STATUS ) String messageStatus
    ) {
```